packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -42,7 +41,7 @@ export const SimpleDocument: FC<Props> = ({
   if (document) {
     //Json object usually don't have enough info to allow us to determine which field to display
     //Unless there is passage pointing to a specific field.
-    const isJsonType = get(document, 'extracted_metadata.file_type') === 'json';
+    const isJsonType = document?.extracted_metadata?.file_type === 'json';
```

Comment:
This now works with this PR. Previously, would fail in lint step.

Reply:
can we get rid of lodash/get altogether?

Reply:
Yes, we can. But best to handle as separate PR -- this was just meant to be a demonstration of something that previously failed but now works with this PR.